src/Board.js

```diff
@@ -23,13 +25,14 @@ export default function Board({
   ];
 
   const [turn, setTurn] = useState(blankBoard);
+  const [hasCoin, setHasCoin] = useState(false);
```

Comment:
A problem I anticipate is that there are cards and shapes where it's possible to get two coins with a single shape. Using a boolean here only allows one coin per shape

Reply:
A counter would fix this